compoments/report-repair/repair-description.js

```diff
@@ -136,6 +137,45 @@ const RepairDescription = ({handleChange, values}) => {
     return errorSummaryTextAndLocation;
   }
 
+  const ImageUploadRender = () => {
```

Comment:
I feel it would be better if this was extracted from the repair-description
- to avoid nested components
- to remove the responsibility of this class to understand the feature flag (checking feature flag could be within `route` which would provide `repair-description` with either this component or a component that renders just  an empty string.)

Reply:
After re-reading the PR description (this is not easy to extract), I still feel that the feature flag could be extracted and this component passed a boolean that indicates whether to show the image upload or not.

Reply:
I've added global functions in a helper file so it now calls isMvpReleaseVersion()